test/e2e/parallel/registration_service_test.go

```diff
@@ -779,16 +776,52 @@ func assertGetSignupReturnsNotFound(t *testing.T, await wait.Awaitilities, beare
 	invokeEndpoint(t, "GET", route+"/api/v1/signup", bearerToken, "", http.StatusNotFound)
 }
 
+// waitForUserSignupReadyInRegistrationService waits and checks that the UserSignup is ready according to registration service /signup endpoint
+func waitForUserSignupReadyInRegistrationService(t *testing.T, registrationServiceURL, name, bearerToken string) map[string]interface{} {
+	t.Logf("waiting and verifying that UserSignup '%s' is ready according to registration service", name)
+	var mp, mpStatus map[string]interface{}
+	err := k8swait.Poll(time.Second*5, time.Second*60, func() (done bool, err error) {
```

Comment:
one second should be fine 
```suggestion
	err := k8swait.Poll(time.Second, time.Second*60, func() (done bool, err error) {
```